setup.cfg

```diff
@@ -70,7 +71,7 @@ test =
     mock
     pytest
     pytest-cov
-    requests ~= 2.20
+    responses
```

Comment:
this is a new beast to me!  I have installed system wide python3-responses (0.9.0-2), but then
```
___________________________________________________________________________________________ ERROR collecting dandi/tests/test_utils.py ____________________________________________________________________________________________
dandi/tests/test_utils.py:6: in <module>
    import responses
/usr/lib/python3/dist-packages/responses.py:11: in <module>
    from collections import namedtuple, Sequence, Sized
<frozen importlib._bootstrap>:1039: in _handle_fromlist
    ???
/usr/lib/python3.8/collections/__init__.py:49: in __getattr__
    warnings.warn("Using or importing the ABCs from 'collections' instead "
E   DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
```
so we might need to allow for that warning?  With `pip install --upgrade responses` I got 0.9.0 and then it worked. .. I would till prefer to allow for earlier one to be allowed and suppress that DeprecationWarning -- we know that they fixed it up

Reply:
I've configured pytest to ignore DeprecationWarnings from responses now.

Reply:
great, thanks!